roles/st2mistral/tasks/main.yml

```diff
@@ -40,11 +29,9 @@
     src: init_mistral_db.SQL.j2
     dest: /etc/mistral/init_mistral_db.SQL
   register: mistral_deploy_database_init_script
-  notify:
-    - restart mistral
   tags: st2mistral
 
-- name: Initiate database
+- name: Initiate mistral database
   become: yes
   become_user: postgres
   shell: psql < /etc/mistral/init_mistral_db.SQL
```

Comment:
Here we have a problem when user considering to change any of `st2mistral_` database settings in future, - the `SQL` script will not be run twice. That means, DB is not created.

We can workaround with additional lock files, but https://github.com/StackStorm/ansible-st2/issues/95 should solve it in a right way.